frontend/scripts/buildCache.ts

```diff
@@ -1,24 +1,31 @@
 /**
  * Codegens API typings
  */
+import * as S3 from 'aws-sdk/clients/s3'
 import Axios from 'axios'
 import 'isomorphic-fetch'
 import { seq } from 'promise-seq'
 import { CONFIG } from '../src/config/config'
 import { DATASETS } from '../src/constants/datasets'
 import { Method } from '../src/constants/datatypes'
 import {
-  getAdequacies, getCensusData, getRepresentativePoints
+  getAdequacies, getCensusData, getRepresentativePoints, getStaticAdequacyUrl, getStaticDemographicsUrl, getStaticRPUrl
 } from '../src/services/api'
 import { safeDatasetHint } from '../src/utils/formatters'
+
 const METHODS: Method[] = ['driving_time', 'straight_line']
 
+const s3Bucket = 'https://s3-us-west-2.amazonaws.com/encompass-public-data/'
+const s3 = new S3()
+const uploadToS3 = true
+const buildCache = true
```

Comment:
what do these do?

Reply:
we only send the files to s3 if uploadToS3 is true and only check that the backend is saving the cache if buildCache is True

Reply:
I mean, why aren't these in the config?